.env

```diff
@@ -1,8 +1,8 @@
-VIRTUAL_HOST=localhost3df
+VIRTUAL_HOST=localhost
 USE_TLS=false
```

Comment:
Videoroom behavior when `USE_TLS` is set to `false` is this same as if it would be unset

Reply:
I have put `USE_TLS` env in the commented environmental variables section